pynamodb/attributes.py

```diff
@@ -517,8 +520,12 @@ def _get_deserialize_class(cls, key, value):
             return cls._get_attributes().get(key)
         return _get_class_for_deserialize(value)
 
+
 def _get_value_for_deserialize(value):
-    return value[list(value.keys())[0]]
+    key = list(value.keys())[0]
```

Comment:
`six.iterkeys` if we want to support py2 and py3. I don't think performance is a real concern here though, as this always only has one element.